uniffi/src/bindings/kotlin/templates/RustBufferHelpers.kt

```diff
@@ -170,13 +181,26 @@ fun String.Companion.lift(ptr: Pointer): String {
     }
 }
 
+fun<T> lowerSequence(v: List<T>, lowersIntoSize: (T) -> Int, lowerInto: (T, ByteBuffer) -> Unit): RustBuffer.ByValue {
+    return lowerIntoRustBuffer(v, { v -> lowersIntoSizeSequence(v, lowersIntoSize) }, { v, buf -> lowerIntoSequence(v, buf, lowerInto) })
+}
+
+fun<T> lowersIntoSizeSequence(v: List<T>, lowersIntoSize: (T) -> Int): Int {
+    var len = v.size.toLong().lowersIntoSize()
```

Comment:
I noticed you chose a 64bit value to hold the size, I remember I chose a 32bit one for strings, I wonder if we should be consistent (I have no opinion on whether both should be 4 bytes or 8 bytes, just an observation)

Reply:
Oh oh also on a related note, and this is something I did too, we use `Long` Kotlin, but Long is signed. I know it is practically impossible for someone to pass an in-memory value that might cross that limit, but just thought of that. ( I make the same mistake with Strings)